documentation/modules/configuring/proc-managing-storage-affinity-node-pools.adoc

```diff
@@ -109,17 +109,17 @@ spec:
 ----
 
 . Apply the node pool configuration.
-. Check the status of the deployment and wait for the pods in the node pools to be created and have a status of `READY`.
+. Check the status of the deployment and wait for the pods in the node pools to be created and have a status of `Running`.
```

Comment:
I don't think this is correct. The status running does not mean they are ready. We want the pods to be Ready => i.e. the `READY` column should be `1/1` (Or 2/2 etc. in case of Entitty Operator and so on). This should apply here as well as in the other cases.